es5-sham.js

```diff
@@ -65,7 +65,7 @@ if (!Object.getPrototypeOf) {
         /* eslint-enable no-proto */
         if (proto || proto === null) {
             return proto;
-        } else if (object.constructor) {
+        } else if (typeof object.constructor === 'function') {
```

Comment:
Correct. I saw `typeof` checks in other places so wasn't sure your chance on it.